### ISSUE TITLE
'documentation' is not a valid reporter type

### ISSUE THREAD

**Issue by Annih** [inspec/inspec#2609]
### Description

Our kitchen-tests are now failing with the following error:
> Kitchen::ActionFailed: Failed to complete #verify action: ['documentation' is not a valid reporter type.]

This seems to be related to the reporter feature (#2464).

### InSpec and Platform Version

Failing with Inspec *v1.51.15* on CentOS 7.2

### Stacktrace

```19:42:09 Kitchen::ActionFailed: Failed to complete #verify action: ['documentation' is not a valid reporter type.]
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/inspec-1.51.15/lib/inspec/reporters.rb:21:in `render'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/inspec-1.51.15/lib/inspec/runner.rb:109:in `block in render_output'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/inspec-1.51.15/lib/inspec/runner.rb:108:in `each'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/inspec-1.51.15/lib/inspec/runner.rb:108:in `render_output'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/inspec-1.51.15/lib/inspec/runner.rb:132:in `run_tests'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/inspec-1.51.15/lib/inspec/runner.rb:102:in `run'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/kitchen-inspec-0.20.0/lib/kitchen/verifier/inspec.rb:94:in `call'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/test-kitchen-1.20.0/lib/kitchen/instance.rb:443:in `block in verify_action'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/test-kitchen-1.20.0/lib/kitchen/instance.rb:536:in `synchronize_or_call'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/test-kitchen-1.20.0/lib/kitchen/instance.rb:498:in `block in action'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/test-kitchen-1.20.0/lib/kitchen/instance.rb:497:in `action'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/test-kitchen-1.20.0/lib/kitchen/instance.rb:435:in `verify_action'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/test-kitchen-1.20.0/lib/kitchen/instance.rb:368:in `block in transition_to'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/test-kitchen-1.20.0/lib/kitchen/instance.rb:367:in `each'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/test-kitchen-1.20.0/lib/kitchen/instance.rb:367:in `transition_to'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/test-kitchen-1.20.0/lib/kitchen/instance.rb:157:in `verify'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/test-kitchen-1.20.0/lib/kitchen/instance.rb:186:in `block in test'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/test-kitchen-1.20.0/lib/kitchen/instance.rb:182:in `test'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/retryable-kitchen-0.1.6/lib/retryable_kitchen/tasks.rb:18:in `launch'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/retryable-kitchen-0.1.6/lib/retryable_kitchen/tasks.rb:59:in `block (3 levels) in define_tasks'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/parallel-1.12.0/lib/parallel.rb:484:in `call_with_index'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/parallel-1.12.0/lib/parallel.rb:455:in `process_incoming_jobs'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/parallel-1.12.0/lib/parallel.rb:437:in `block in worker'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/parallel-1.12.0/lib/parallel.rb:428:in `fork'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/parallel-1.12.0/lib/parallel.rb:428:in `worker'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/parallel-1.12.0/lib/parallel.rb:419:in `block in create_workers'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/parallel-1.12.0/lib/parallel.rb:418:in `each'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/parallel-1.12.0/lib/parallel.rb:418:in `each_with_index'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/parallel-1.12.0/lib/parallel.rb:418:in `create_workers'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/parallel-1.12.0/lib/parallel.rb:358:in `work_in_processes'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/parallel-1.12.0/lib/parallel.rb:264:in `map'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/parallel-1.12.0/lib/parallel.rb:272:in `map_with_index'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/retryable-kitchen-0.1.6/lib/retryable_kitchen/tasks.rb:58:in `block (2 levels) in define_tasks'
19:42:09 /home/jenkins/.gem/ruby/2.4.0/gems/rake-12.3.0/exe/rake:27:in `<top (required)>'
19:42:09 /bin/bundle:22:in `load'
19:42:09 /bin/bundle:22:in `<main>'
```

*Cc.* @aboten

**Comment by jquick** [comment]
@Annih - Hey thanks for this post, this should be resolved in v1.51.18. Please let me know if your still having issues under that version. Thanks!

**Comment by jquick** [comment]
Hey @Annih - The new versions of 1.x or 2.x should not have this issue. Feel free to open a issue or reach out in the community slack if your still having issues after upgrading!
